src/openms_gui/source/VISUAL/SpectraViewWidget.cpp

```diff
@@ -55,11 +54,68 @@ namespace OpenMS
     return out;
   }
 
+
+  QList<QVariant> vecToList(const std::vector<Size>& in)
```

Comment:
this looks familiar... did you implement that somewhere else already?

Reply:
the reverse operation. Just above :) ?